_sass/template/partials/_pdf-slides.scss

```diff
@@ -193,23 +193,25 @@ $print-slides: true !default;
         }
     }
 
-    @if $print-sidebar {
+    @if $pdf-sidebar {
 
         .slides {
             column-count: 3;
-            column-gap: $grid-unit-width;
-            margin-outside: -($sidebar-text-width + $sidebar-margin-inside); // -60mm;
+            column-gap: $sidebar-gap-width;
+            margin-outside: -($sidebar-content-width + $sidebar-gap-width);
             margin-inside: 0;
+            width: $content-width;
 
             .figure {
-                width: $sidebar-text-width;
+                display: inline-block;
+                box-sizing: border-box;
 
                 // When there are only two slides (a summary and two figures)
                 // we shift those inside to avoid the empty third column.
                 // This CSS selects the second and third of three siblings.
                 &:nth-child(2):nth-last-child(2),
                 &:nth-child(3):nth-last-child(1) {
-                    margin-outside: $sidebar-text-width + $sidebar-margin-inside;
+                    // margin-outside: $sidebar-content-width + $sidebar-gap-width;
```

Comment:
If this is commented out, the selector is empty - can we remove it?

Reply:
Hmm, actually I need to do more testing here before we merge.

Reply:
After more testing, this and other issues resolved in [13ab587](https://github.com/electricbookworks/electric-book/pull/664/commits/13ab5875302d269f831aa31761852088128bd3fe).